### PR TITLE
style(ui): general improvements

### PR DESCRIPTION
**Pull request by dy-tea**

### CODE REVIEW

**Comment by krypt0nn** [comment]
Disable formatting on save. Rejected for now.

File: src/ui/components/picture_carousel.rs [hunk 1]
```diff
@@ -52,38 +55,67 @@ impl SimpleAsyncComponent for PictureCarousel {
...
-                set_height_request: CardComponent::default().height,
+        gtk::Overlay {
+            gtk::Box {
+                set_orientation: gtk::Orientation::Vertical,
+                set_spacing: 16,
+
+                model.pictures.widget() {
+                    // 16:9 image
+                    set_height_request: CardComponent::default().height,
+                    set_width_request: (CardComponent::default().height as f32 * 1.777777777777) as i32,
```
Comment (krypt0nn) [inline src/ui/components/picture_carousel.rs:66]: `* 16.0 / 9.0` ?

File: src/ui/windows/main_window/store_page.rs [hunk 2]
```diff
@@ -157,6 +157,8 @@ impl SimpleAsyncComponent for StorePage {
 
                 self.show_game_page = true;
             }
+
+            StorePageInput::Activate => {}
```
Comment (krypt0nn) [inline src/ui/windows/main_window/store_page.rs:161]: Why did you return this event back when it does *nothing*?

**Comment by krypt0nn** [comment]
Considering amount of changes it would be easier to rebase to the latest commit and make a new commit with everything you want.

File: src/ui/windows/main_window/store_page.rs (hunk 2 above)
Comment (dy-tea) [reply src/ui/windows/main_window/store_page.rs:161]: It doesn't do nothing, it runs the code below that which updates the back button showing when switching between the store and library.

**Pull request merged**